dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -353,14 +353,14 @@
 
             // refresh MethodConfigs
             List<MethodConfig> methodConfigs = this.getMethods();
-            if (methodConfigs != null && methodConfigs.size() > 0) {
+            if (methodConfigs != null && !methodConfigs.isEmpty()) {
```

Comment:
**Suggestion**: Using '!methodConfigs.isEmpty()' instead of 'methodConfigs.size() > 0' is more idiomatic and readable. Good job on this change. <br> **Code Suggestion**: 
 ```
 The change is correctly applied as suggested.
 ```